docs/architecture/grpc-for-wcf-developers/encryption.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Encryption and network security - gRPC for WCF Developers
+description: Some notes on network security and encryption in gRPC
+author: markrendle
+ms.date: 09/02/2019
+---
+
+# Encryption and network security
+
+WCF's network security model is extensive and complex, including transport-level security using HTTPS or TLS-over-TCP, and message-level security using the WS-Security specification to encrypt individual messages.
+
+gRPC leaves secure networking to the underlying HTTP/2 protocol, which can be secured using regular TLS certificates.
+
+Web browsers insist on using TLS connections for HTTP/2, but most programmatic clients, including .NET's `HttpClient`, can use HTTP/2 over unencrypted connections. `HttpClient` *does* require encryption by default, but you can override this using an <xref:System.AppContext> switch.
```

Comment:
Mention that this switch must be set before the HttpClient/HttpClientHandler (or GrpcChannel if no HttpClient is explicitly specified) is created.